cockatrice/src/sequenceEdit/secuenceedit.cpp

```diff
@@ -0,0 +1,150 @@
+#include "secuenceedit.h"
+#include "../settingscache.h"
+#include <QLineEdit>
+#include <QPushButton>
+#include <QHBoxLayout>
+#include <QEvent>
+#include <QKeyEvent>
+#include <QToolTip>
+
+SecuenceEdit::SecuenceEdit(QString _shorcutName, QWidget *parent) : QWidget(parent)
+{
+    shorcutName = _shorcutName;
+    currentKey = 0;
+    maxKeys = 4;
+    keys = 0;
+    valid = false;
+
+    lineEdit = new QLineEdit(this);
+    clearButton = new QPushButton("", this);
+    defaultButton = new QPushButton("", this);
+
+    lineEdit->setMinimumWidth(100);
+    clearButton->setMaximumWidth(lineEdit->height());
+    defaultButton->setMaximumWidth(lineEdit->height());
```

Comment:
What if each grouping of shortcuts was it's own tab inside that panel?

On Tue, Aug 18, 2015, 5:37 PM Zach <notifications@github.com> wrote:

> In cockatrice/src/sequenceEdit/secuenceedit.cpp
> <https://github.com/Cockatrice/Cockatrice/pull/1382#discussion_r37356854>:
>
> > +
> > +SecuenceEdit::SecuenceEdit(QString _shorcutName, QWidget *parent) : QWidget(parent)
> > +{
> > +    shorcutName = _shorcutName;
> > +    currentKey = 0;
> > +    maxKeys = 4;
> > +    keys = 0;
> > +    valid = false;
> > +
> > +    lineEdit = new QLineEdit(this);
> > +    clearButton = new QPushButton("", this);
> > +    defaultButton = new QPushButton("", this);
> > +
> > +    lineEdit->setMinimumWidth(100);
> > +    clearButton->setMaximumWidth(lineEdit->height());
> > +    defaultButton->setMaximumWidth(lineEdit->height());
>
> Same issue here on a larger mac
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Cockatrice/Cockatrice/pull/1382/files#r37356854>.
>